settings.yaml

```diff
@@ -78,7 +78,8 @@ nodestore:
   database: simple
 
 qdrant:
-  path: local_data/private_gpt/qdrant
```

Comment:
I'd keep the default setting using the disk-based local qdrant, instead of requiring a local qdrant service